ee/tuf/library_lookup.go

```diff
@@ -13,6 +13,7 @@ import (
 	"github.com/kolide/launcher/pkg/launcher"
 	"github.com/kolide/launcher/pkg/traces"
 	"github.com/peterbourgon/ff/v3"
+	"github.com/spf13/pflag"
```

Comment:
I feel mixed about bringing in a new library, and I think we should be somewhat intentional about it.

I think the problem we have with `ff` and the standard `flags` is that they throw errors on unknown flags. I assume `pflag` handles that. Does pflag also support reading from a config file?

I think I'm game for us moving everything to pflag, and starting here. Or maybe developing our own. But I think we should _not_ use both ff and pflag in this same file...

Reply:
Correct, I had to bring in [pflag](https://pkg.go.dev/github.com/spf13/pflag) because `flag`, and `ff` because it uses `flag` for command-line args, will not parse args if they contain unknown flags.

`pflag` does not support reading from a config file. The other option I considered was [go-flags](https://pkg.go.dev/github.com/jessevdk/go-flags), but this also doesn't have support for config file. There weren't any other packages I could find that would handle unknown flags appropriately, let alone do that plus support a config file. (We actually already had pflag in go.sum, as well.)

Reply:
I can't decide if we should look at rolling our own. Or finding some other way to read a config and merge, or what. I don't love any of it, really.

Maybe the hacky way is to read the config into an array, and append flags, and parse the whole thing.

Reply:
I guess I could live with this, but using two flag libraries definitely feels like it raises the messy factor

Reply:
I'm game for moving everything to pflag or another option at some point, but I am hesitant to do that now since changing command-line flag parsing is risky and I'd rather avoid adding more risk to https://github.com/kolide/launcher/issues/954.

Reply:
Yeah... I agree we shouldn't do it in _this_ PR. But I think we should figure out where we want to go, and slowly move there. 

I haven't enjoyed the various tools from spf13, so I'm initially skeptical of this one

Reply:
Wrote up a quick issue: https://github.com/kolide/launcher/issues/1513 -- feel free to edit.

I chose this one over `go-flags` because it's more widely used and has been updated more recently. I'm happy to trial `go-flags` for this usage if you'd like.

Reply:
Meh. We can evaluate under #1513. This is probably fine here